Octokit.Tests.Integration/Clients/PullRequestsClientTests.cs

```diff
@@ -225,6 +226,20 @@ public PullRequestsClientTests()
     }
 
     [IntegrationTest]
+    public async Task CanBeMergedWithSquashCommit()
+    {
+        await CreateTheWorld();
+
+        var newPullRequest = new NewPullRequest("a pull request", branchName, "master");
+        var pullRequest = await _fixture.Create(Helper.UserName, _context.RepositoryName, newPullRequest);
+
+        var merge = new MergePullRequest { CommitMessage = "fake message", CommitTitle = "fake title", Squash = true };
+        var result = await _fixture.Merge(Helper.UserName, _context.RepositoryName, pullRequest.Number, merge);
+
+        Assert.True(result.Merged);
```

Comment:
Could we assert something here about the commit title or message versus what we actually committed, to indicate this was squashed?